src/sharedQueue.h

```diff
@@ -0,0 +1,142 @@
+/**
+ * \file	serial.h
+ * \author	Lucas Mongrain
```

Comment:
I appreciate, but I didn't make this one, only modify it a little: https://stackoverflow.com/questions/36762248/why-is-stdqueue-not-thread-safe

Reply:
Change it with a direct link to the forum